CHANGELOG.md

```diff
@@ -1,9 +1,28 @@
-# waSCC Host Changelog
+# waSCC Host Change Log
 
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+_The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)_
+
+## [0.12.0] - ?? AUG ??
+
+This version corresponds to the project milestone 0.12
+
+### Added
+
+* _Host Labels_ - You can now add arbitrary key/value pairs to a host manifest (when feature enabled) or via the `set_label` function (when `lattice` is enabled). These labels are discoverable via lattice host probe query. The host runtime will automatically add the appropriate values for the following reserved labels which cannot be overridden:
+
+    * hostcore/os
+    * hostcore/osfamily
+    * hostcore/arch
```

Comment:
In code these are `hostcore.arch` etc.

Reply:
Good catch! I changed my mind on the slash halfway through the code. I'll update the changelog